docs/upgrading-solidus/v4.6.mdx

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_position: -4.6
+---
+
+import MinimalRequirements from '@site/src/theme/MinimalRequirements';
+
+# Solidus v4.6 (2025-09-09)
+
+<MinimalRequirements ruby="3.1" rails="7.0" />
+
+We're excited to announce the release of Solidus v4.6.0, packed with improvements across all components of the framework. This release focuses on enhanced email handling, reverse charge VAT support, improved admin UI components, and better Rails compatibility.
```

Comment:
```suggestion
We're excited to announce the release of Solidus v4.6.0, packed with improvements across all components of the framework. This release focuses on event-based transactional email configuration, reverse charge VAT support, and improved admin UI components
```

Two suggested tweaks:

- "enhanced email handling" seemed a bit general
- "better Rails compatibility" should be invisible, and maybe isn't worth highlighting up here. Your call!